gytheio-commons/src/main/java/org/gytheio/content/handler/ContentReferenceHandler.java

```diff
@@ -79,7 +80,18 @@
      * @throws ContentIOException
      */
     public long putInputStream(InputStream sourceInputStream, ContentReference targetContentReference) throws ContentIOException;
-    
+
+    /**
+     * Writes the given source file into the given target content reference.
+     * <p>
```

Comment:
Minor note, trailing <p>?

Reply:
I assume you mean trailing <p> ? 

Done (I've removed it) - thanks for the feedback.